runtimes/web/src/compositor.ts

```diff
@@ -103,12 +115,17 @@ export class WebGLCompositor {
 
         // Setup shaders
         const program = gl.createProgram();
+
+        if(!program) {
+            throw new Error('WebGLCompositor: cannot create program');
```

Comment:
Are these new null checks to appease the type checker? For most of these cases can we signal that it won't be null?

```
const program = gl.createProgram()!;
```